share/util/shr_reprosum_mod.F90

```diff
@@ -604,7 +623,9 @@ subroutine shr_reprosum_calc (arr, arr_gsum, nsummands, dsummands,     &
       else
          mpi_comm = MPI_COMM_WORLD
       endif
+#ifndef EAMXX_STANDALONE
```

Comment:
Was this not working in EAMxx standalone testing? We do have timers on in hommexx, so I wonder why these would not work...

Reply:
resolved in person